mackup/main.py

```diff
@@ -87,8 +87,14 @@ def printAppHeader(app_name):
         # Check the env where the command is being run
         mckp.check_for_usable_backup_env()
 
+        applications = sorted(mckp.get_apps_to_backup())
+        
+        # To allow for specific applications to be backed up, we replace the full list with only the valid ones from the command line
+        if args["<application>"]:
+            applications = list(set(args["<application>"]) & set(applications))
```

Comment:
should it fail here if any explicit application is not recognized?
silent ignoring them seems dangerous, as it would imply everything went fine

Reply:
it should also be sorted again, to maintain the same behaviour